estate_account/models/estate_property.py

```diff
@@ -0,0 +1,25 @@
+from odoo import Command, models
+
+
+class EstateProperty(models.Model):
+    _inherit = "estate.property"
+
+    def estate_sold(self):
+        self.env["account.move"].create({
+            "name": "Test",
+            "move_type": "out_invoice",
+            "partner_id": self.env["estate.property"].buyer_id,
+            "line_ids": [
+                Command.create({
+                    "name": "Taxe",
+                    "quantity": "1",
+                    "price_unit": str(float(self.selling_price) * 0.06)
+                }),
+                Command.create({
+                    "name": "Administrative fees",
+                    "quantity": "1",
+                    "price_unit": "100"
```

Comment:
```suggestion
                    "quantity": 1,
                    "price_unit": 100,
```
no need for string here, as the values are int or float